src/main/java/sirius/tagliatelle/compiler/RawHandler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.tagliatelle.compiler;
+
+import sirius.kernel.di.std.Register;
+import sirius.tagliatelle.emitter.Emitter;
+import sirius.tagliatelle.emitter.RawEmitter;
+
+/**
+ * Handles <tt>{@literal @}raw {}</tt>.
```

Comment:
the javadoc states this handles `@raw {}` but the code looks like the expected syntax is `@raw() {}`